test/stock_takings/stock_takings.test.ts

```diff
@@ -0,0 +1,537 @@
+  user.clientAccount = process.env.SYSTEM_TEST_CLIENT_ACCOUNT_ID || user.clientAccount
+  user.apiKey = process.env.SYSTEM_TEST_API_KEY || user.apiKey
+}
+
+const legacyId = '4564'
+const stockTakingId = '1337'
+const mockStockTaking = { status: 'started', result: { items: [{ code: '1337', amount: 1337 }] } } as StockTaking
+const mockMsg = `Deleted stockTaking ${stockTakingId}`
+
+const mock = new MockAdapter(axios)
+afterEach(() => {
+  mock.reset()
+})
+
+describe('v0: StockTakings', () => {
+  it('retrieves all stockTakings', async () => {
+    if (process.env.SYSTEM_TEST !== 'true') {
+      mock.onPost('https://api.tillhub.com/api/v0/users/login').reply(function (config) {
+        return [
+          200,
+          {
+            token: '',
+            user: {
+              id: '123',
+              legacy_id: legacyId
+            }
+          }
+        ]
+      })
+
+      mock
+        .onGet(`https://api.tillhub.com/api/v0/stock_takings/${legacyId}`)
+        .reply(function (config) {
+          return [
+            200,
+            {
+              count: 1,
+              results: [{}]
+            }
+          ]
+        })
+    }
+
+    const options = {
+      credentials: {
+        username: user.username,
+        password: user.password
+      },
+      base: process.env.TILLHUB_BASE
+    }
+
+    const th = new TillhubClient()
+
+    th.init(options)
+    await th.auth.loginUsername({
+      username: user.username,
+      password: user.password
+    })
+
+    const stockTakings = th.stockTakings()
+
+    expect(stockTakings).toBeInstanceOf(StockTakings)
+
+    const { data } = await stockTakings.getAll()
+
+    expect(Array.isArray(data)).toBe(true)
+  })
+
+  it('retrieves one stockTaking', async () => {
+    if (process.env.SYSTEM_TEST !== 'true') {
+      mock.onPost('https://api.tillhub.com/api/v0/users/login').reply(function (config) {
+        return [
+          200,
+          {
+            token: '',
+            user: {
+              id: '123',
+              legacy_id: legacyId
+            }
+          }
+        ]
+      })
+
+      mock
+        .onGet(`https://api.tillhub.com/api/v0/stock_takings/${legacyId}/${stockTakingId}`)
+        .reply(function (config) {
+          return [
+            200,
+            {
+              count: 1,
+              results: [{}]
+            }
+          ]
+        })
+    }
+
+    const options = {
+      credentials: {
+        username: user.username,
+        password: user.password
+      },
+      base: process.env.TILLHUB_BASE
+    }
+
+    const th = new TillhubClient()
+
+    th.init(options)
+    await th.auth.loginUsername({
+      username: user.username,
+      password: user.password
+    })
+
+    const stockTakings = th.stockTakings()
+
+    expect(stockTakings).toBeInstanceOf(StockTakings)
+
+    const { data } = await stockTakings.get(stockTakingId)
+
+    expect(Array.isArray(data)).toBe(false)
```

Comment:
I think this line doesn't make much sense to me.

Maybe it would make more sense to pass `mockStockTaking` to the 200 response' results, i.e. `results:[mockStockTaking]` and then assert the response like so (you're already doing so in other places):
```suggestion
    expect(data).toEqual(mockStockTaking)
```

Reply:
done